cloudinit/distros/bsd_util.py

```diff
@@ -11,6 +11,7 @@
 # The module comes with a more naive parser, but is able to
 # preserve these if blocks.
 
+
```

Comment:
where is the comment?

Reply:
hum, here? https://github.com/canonical/cloud-init/pull/62/files/0e6770765631a44357e2ccc24201a3a96976ac71..6acc2c9b8cc1e27583e1707b8a1412925df75e92#diff-105f5fa921e17a90914b0dd9237d673cR7-R12